src/caret_analyze/runtime/node.py

```diff
@@ -481,10 +489,3 @@ def is_target_publisher(publishers: Publisher):
             return match
 
         return Util.find_one(is_target_publisher, self._publishers)
-
-    def get_timer(self, topic_name: str) -> Timer:
-        # TODO(hsgwa): fix argument type.
-        if not isinstance(topic_name, str):
-            raise InvalidArgumentError('Argument type is invalid.')
-
-        return Util.find_one(lambda x: x.topic_name == topic_name, self._timers)
```

Comment:
I think this function raises exception when it is called because Timer does not have topic_name. So it is preferable to remove this function.

Reply:
@isp-uetsuki 
Thank you for your information. I found out why it was deleted. It was implemented incorrectly, and since the problem was not apparent until now, there seems to be no problem with deleting it.